recovery_behavior/plugins/action/recover_pose_action.cpp

```diff
@@ -0,0 +1,86 @@
+// Licensed under MIT. See LICENSE file. Copyright Carologistics.
```

Comment:
removce long license header, keep line 1

Reply:
fixed, pls refer to commit - dce5e3e